### PR TITLE
code updated for dynamic user data

### PR DESCRIPTION
**Pull request by prabhamanju**
hi plz review the code and let me know your valuable feedback...!! thanks in advance.

### CODE REVIEW

**Review by HYFar — COMMENTED** [review]
I'd suggest to use your personal data for the mock user so we have a real example to work with

**Comment by JoeCamacho** [comment]
Not sure if we need this data for the moment since the idea is to take the data from the model

**Comment by prabhamanju** [comment]
> I'd suggest to use your personal data for the mock user so we have a real example to work with

Ok thanks!! working more on the same file .... I will keep your suggestion in.

**Comment by prabhamanju** [comment]
> Not sure if we need this data for the moment since the idea is to take the data from the model

yeah Joel !! you are right but as of now we don't have server program ready so I used that of testing purpose. But i am working more on this to enhance the code.

**Comment by archived-m** [comment]
The mock user is actually a great idea until you connect the client and server-side.
Well done Manju :)

File: client/src/components/MocUser.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// Fake contact data for testing purposes.
+
+const MocUser = [
```
Comment (Toinne) [inline client/src/components/MocUser.js:3]: Try and make names plural if they can contain multiple elements. Typically for array and objects.

**Review by Toinne — APPROVED** [review]
Great stuff ! Cool that you are importing your MocUser !

**Pull request closed without merging**